tests/test_vision_explanation.py

```diff
@@ -139,15 +142,22 @@ def test_vision_explain_loadmodel():
 
     # run the main function for saliency map generation
     device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
-    model = _get_instance_segmentation_model(5)
+    model = _get_instance_segmentation_model(NUM_CLASSES)
     model.load_state_dict(torch.load(modelpath, device))
     model.to(device)
 
+    if use_transforms:
+        resize_transforms = torchvision.transforms.Resize(size=TRANSFORM_SIZE)
```

Comment:
Curious, what kind of resizing is being done here and why?

Is this the resizing for reducing memory footprint of images? If yes, should this be enforced with automl models, or I wonder if users should be able to set a flag on whether they'd like their images to be compressed to low res rather than we doing it under the hood without them perhaps knowing ... ?

Reply:
This is just the test case - there are various transformations that could be specified here.  AutoML has these transforms from torchvision hence to be compatible with their models we need to run them here.  This resize transform here is just an example.